test/lit/wat-annotations.wast

```diff
@@ -1,10 +1,38 @@
 ;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
 
-;; RUN: wasm-opt -all %s -S -o - | filecheck %s
+;; RUN: wasm-opt -all             %s -S -o - | filecheck %s
+;; RUN: wasm-opt -all --roundtrip %s -S -o - | filecheck %s --check-prefix=BINARY
```

Comment:
Sometimes we use `RTRIP` as the check prefix so it has the same number of characters as `CHECK`.

Reply:
Done.